test/org/redhatcop/util/files/params/notifications-build.params

```diff
@@ -0,0 +1,5 @@
+APPLICATION_NAME=notifications-test
+NAMESPACE=pipeline-library-test
+SOURCE_REPOSITORY_URL=https://github.com/etsauer/pipeline-library.git
+SOURCE_REPOSITORY_REF=shared-lib
```

Comment:
Same as above, default is `master` which will be true once merged.

Reply:
yes